### PR TITLE
French (fr-CM): Adding FCFA currency :: cameroon and Central Africa

### PR DESCRIPTION
**Pull request by bmohamet**
Adding FCFA currency for cameroon and Central Africa.

### CODE REVIEW

File: rails/locale/fr-CM.yml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  number:
+    currency:
+      format:
+        delimiter: " "
+        format: "%n %u"
+        precision: 2
+        separator: ","
+        significant: false
+        strip_insignificant_zeros: false
+        unit: "FCFA"
```
Comment (bmohamet) [inline rails/locale/fr-CM.yml]: Currency XAF added.

**Comment by digitalfrost** [comment]
Fails with

```
1) a rails-i18n rails/locale/fr-CM.yml locale file should be named like top level namespace
     Failure/Error: it { should be_named_like_top_level_namespace }
       expected "rails/locale/fr-CM.yml" to be named like top level namespace
     # ./spec/unit/translation_spec.rb:13:in `block (3 levels) in <top (required)>'
```

File: rails/locale/fr-CM.yml [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+fr:
```
Comment (digitalfrost) [inline rails/locale/fr-CM.yml]: Should be 
```
fr-CM:
```

**Comment by digitalfrost** [comment]
Closing this as stale

**Pull request closed without merging**